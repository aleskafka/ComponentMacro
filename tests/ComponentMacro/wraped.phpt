<?php

/**
 * Test: Latte\Engine: {include file}
 */

require __DIR__ . '/../bootstrap.php';

$latte = new Latte\Engine;
$latte->setTempDirectory(TEMP_DIR);
ComponentMacro::install($latte, __DIR__ . '/components');

matchFile(
	__DIR__ . '/expected/wraped.html',
	$latte->renderToString(__DIR__ . '/templates/navigation.wraped.latte', ['title' => 'Hello', 'navigation' => ['John']])
);

