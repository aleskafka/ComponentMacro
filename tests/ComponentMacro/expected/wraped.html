
<div class="header-wraped"><div class="header">
	<h1>Hello</h1>
	<ul class="header-navigation">
		<li>John</li>
	</ul>
</div>
</div>
